testpar/t_filters_parallel.c

```diff
@@ -9829,9 +9843,26 @@ main(int argc, char **argv)
     dcpl_id = H5Pcreate(H5P_DATASET_CREATE);
     VRFY((dcpl_id >= 0), "DCPL creation succeeded");
 
+    /* Add a space after the HDF5_PARAPREFIX notice from h5_fixname */
```

Comment:
space -> newline

Reply:
A _vertical_ space, though, to be completely correct, a _blank line_ since a newline isn't entirely accurate either.